src/build.ts

```diff
@@ -173,14 +168,46 @@ export class Builder {
             const cfg = await AssetPublisher.getConfig(rawCfg, process.env);
             const kvArgs: KVArgs = Object.assign({ retries: 0 }, cfg.kv);
 
-            const builder = Builder.init();
+            const targetDir = process.cwd();
+
+            let builder = new Builder(targetDir);
+            let manifest;
 
             let webpackConfigPath = BASIC_CFG_PATH;
             if (!options.staticSite) {
                 await builder.buildClient();
+
+                Builder.createWorkerDir(targetDir);
+                manifest = builder.generateManifest(options.assetsDir);
+
                 webpackConfigPath = WORKER_CFG_PATH;
+            } else {
+                // checking static site template
+                const templatePath = 'azion/cells-site-template'
```

Comment:
Is `'azion/cells-site-template'` a constant that can be used at the `asset-publisher.ts` too?